### PR TITLE
WIP Use MCD once-from instead of coreos-installer in BIP mode

### PR DESCRIPTION
**Pull request by vrutkovs**
This reprovisions bootstrap node into a master in BIP mode

### CODE REVIEW

File: data/data/bootstrap/bootstrap-in-place/files/usr/local/bin/install-to-disk.sh.template [hunk 1]
```diff
@@ -14,9 +14,22 @@ record_service_stage_success
 
 if [ ! -f coreos-installer.done ]; then
   record_service_stage_start "coreos-installer"
+  {{ if .IsOKD -}}
+
+  # Delete bootstrap manifests
+  rm -rf /etc/kubernetes/manifests/*
+
+  # Apply new ignition using start --once-from. This doesn't require a full disk, but may
+  # leave some bootstrap files around
```
Comment (staebler) [inline data/data/bootstrap/bootstrap-in-place/files/usr/local/bin/install-to-disk.sh.template:23]: For OCP, leaving bootstrap files around is unacceptable. I would assume that OKD has the same requirements.

Comment (vrutkovs) [reply data/data/bootstrap/bootstrap-in-place/files/usr/local/bin/install-to-disk.sh.template:23]: Yes, leaving bootstrap files around is not a great idea - but this PR is the best we've got so far, I'll leave a FIXME note

**Comment by vrutkovs** [comment]
No longer required for OKD AI flow, we found a less invasive way

**Pull request closed without merging**